float32 accel_x
float32 accel_y
float32 accel_z

float32 gyro_x
float32 gyro_y
float32 gyro_z

float32 compass_x
float32 compass_y
float32 compass_z
float32 compass_angle

uint64 timestamp
